### PR TITLE
fix(kit): set `pointer-events: none` by readonly state for checkbox

### LINKED ISSUE #3705: 🐞 - pointer-events: none on disabled checkbox
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/stackblitz

### Is this issue blocking you?

Non-Blocking

### Description

Disabled tui-checkbox have pointer-events: none on it, which prevents it from showing hints. Move pointer-events: none on input deeper inside.

### Angular version

11.2.4

### Taiga UI version

2.77

### Which browsers have you used?

- [X] Chrome
- [X] Firefox
- [X] Safari
- [X] Edge

### Which operating systems have you used?

- [X] macOS
- [X] Windows
- [X] Linux
- [X] iOS
- [X] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #3705

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/core/components/primitive-checkbox/primitive-checkbox.component.ts [hunk 1]
```diff
@@ -27,6 +27,10 @@ export class TuiPrimitiveCheckboxComponent {
     @tuiDefaultProp()
     disabled = false;
 
+    @Input()
+    @tuiDefaultProp()
+    readOnly = false;
```
Comment (waterplea) [inline projects/core/components/primitive-checkbox/primitive-checkbox.component.ts:32]: `PrimitiveCheckbox` is already readonly, no need for additional input. Maybe just add `_readonly` class to it in `Checkbox` component.

Comment (splincode) [reply projects/core/components/primitive-checkbox/primitive-checkbox.component.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

File: projects/kit/components/checkbox/checkbox.style.less [hunk 2]
```diff
@@ -19,8 +19,7 @@
         height: @sizeL;
     }
 
-    &._disabled,
```
Comment (nsbarsukov) [inline projects/kit/components/checkbox/checkbox.style.less:22]: Why did you drop `&._disabled` ?

Comment (splincode) [reply projects/kit/components/checkbox/checkbox.style.less:22]: `div[tuiWrapper]` has disabled class and `pointer-events: none`

**Review by waterplea — APPROVED** [review]

**Pull request merged**